packages/backend/src/api/controllers/ForcedTradeOfferController.ts

```diff
@@ -0,0 +1,289 @@
+import { encodeAssetId } from '@explorer/encoding'
+import { AssetId, EthereumAddress, StarkKey, Timestamp } from '@explorer/types'
+import {
+  recoverAddress,
+  solidityKeccak256,
+  solidityPack,
+} from 'ethers/lib/utils'
+
+import {
+  ForcedTradeAcceptRecord,
+  ForcedTradeInitialOfferRecord,
+  ForcedTradeOfferRepository,
+} from '../../peripherals/database/ForcedTradeOfferRepository'
+import {
+  PositionRecord,
+  StateUpdateRepository,
+} from '../../peripherals/database/StateUpdateRepository'
+import { UserRegistrationEventRepository } from '../../peripherals/database/UserRegistrationEventRepository'
+import { ControllerResult } from './ControllerResult'
+
+export class ForcedTradeOfferController {
+  constructor(
+    private offerRepository: ForcedTradeOfferRepository,
+    private stateUpdateRepository: StateUpdateRepository,
+    private userRegistrationEventRepository: UserRegistrationEventRepository
+  ) {}
+
+  async postOffer(
+    offer: Omit<ForcedTradeInitialOfferRecord, 'createdAt' | 'id'>
```

Comment:
Not in scope of this PR, but I would add a requirement for the user to also sign this offer so that only owners of the positions can create offers.

Reply:
Ok, but should we merge this first, as the scope is getting quite broad, it will be harder to do code review, what do you think?